__tests__/__snapshots__/snapshotDiff.test.js.snap

```diff
@@ -52,31 +54,32 @@ exports[`can use contextLines with React components 1`] = `
 + <Component test=\\"my name\\" />
 
 @@ -6,1 +6,1 @@
--    say
-+    my name
+-     say
++     my name
 @@ -9,1 +9,1 @@
--    say
-+    my name
+-     say
++     my name
 @@ -32,1 +32,1 @@
--    say
-+    my name"
+-     say
++     my name"
 `;
 
 exports[`collapses diffs and strips ansi by default 1`] = `
 "Snapshot Diff:
 - First value
 + Second value
 
-@@ -5,8 +5,9 @@
-     some
-     some
-     some
-     some
-     not
-+    so
-     very
-     long
-     script"
+@@ -5,9 +5,10 @@
+      some
+      some
+      some
+      some
+      not
++     so
+      very
+      long
+      script
+  "
```

Comment:
@thymikee This change is from https://github.com/facebook/jest/pull/4520:
* `expand: false` option made sure that strings end with newline
* `expand: true` did not, and in this case it even trimmed the newline at end of expected and received strings!